lips/lip-6.md

```diff
@@ -0,0 +1,79 @@
+* *Validator Owner* is a entity that manages an on-chain Libra account of the member of the Libra Association. The account stores the *Validator Config* resource that contains information about the Validator Node of this Validator Owner. This information includes *consensus public key* (EdDSA public key that is used by others to authenticate consensus messages of this node) and *networking information* (network addresses and static Noise keys for establishing secure connections with this node from other nodes or from clients).
+* *Validator Set* is a copy of Validator Configs of the approved subset of the Validator Owners, it consists of configs currently running the Libra Protocol. The Libra Association collectively decides which Validator Owners' configs are in the set. There is a one-to-one correspondence between the Validator Owners in the Validator Set and the Validator Nodes. The protocol remains safe when at most a third minus one of the Validator Nodes are byzantine.
+* *Validator Operator* ("operator") is an entity that manages its on-chain Libra account and runs a Validator Node on behalf of the Validator Owner who designated this operator to be running the node. There can be one Validator Operator account on-chain that runs two Validator Nodes for two Validator Owners, one node per one owner. It is not allowed to run a single Validator Node per multiple Validator Owners even when the nodes are run by the same Validator Operator. The Validator Operator has the permission to rotate the Validator Configs for its Validator Owners.
+
+# Specification
+## Overview
+Each Validator Owner holds a Libra account with a *Validator Owner role* (see [lip-2: Libra Roles and Permissions](https://github.com/libra/lip/blob/master/lips/lip-2.md)). A Validator Owner designates a Validator Operator to run a Validator Node
+A Validator Owner designates a single Validator Operator account to run a Validator Node. Validator Operator is a Libra account with a *Validator Operator role*. Each Validator Operator may run nodes for multiple Validator Owners, therefore there is a one-to-many relationship between operator accounts and owner accounts. If an operator OP_1 is designated to run Validator Nodes for two Validator Owners V_1 and V_2, the operator must run two distinct instances of Validator Nodes.
+
+Each Validator Owner account stores a consensus configuration that consists of the consensus public key, the networking information (for the Validator Node to connect to other Validator Nodes) and the Full Node networking information (for the public Full Nodes and clients and, under certain circumstances, for other Validator Nodes themselves to connect to this Validator Node). The Validator Operator designated by the Validator Owner is responsible for running a node with this consensus config and is responsible for updating/rotating this config as needed.
```

Comment:
I think we need at least a sentence of background on full nodes in the intro.
"is responsible for" is a little vague.  In the code, ONLY the validator operator may update this information.  In general, we should be precise about who DOESN'T have permission to do something.

Reply:
Agree, is responsible is a little vague. But now members are only "contractually or legally responsible" to run nodes. I guess in the future, they will face some financial losses from their deposits if they don't run the nodes or if their nodes are faulty, but at the moment we only enforce this "contractually".

Let me try to add a bit more on full nodes....